pyproject.toml

```diff
@@ -28,7 +28,8 @@ known_first_party = ["dvc_data"]
 line_length = 79
 
 [tool.pytest.ini_options]
-addopts = "-ra"
+addopts = "-ra --benchmark-skip"
+python_functions = ["bench_*", "test_*"]
```

Comment:
Let's not introduce this complexity with `bench_` prefix. using `test_` is conventional and there is no problem distinguishing them if we just put them into a separate directory like `tests/benchmarks`. Having this magic is only complicating the readability and prevents one from creating special fixtures that could be called `bench_something`.

Reply:
Readability is subjective, I doubt `bench_` instead of `test_` changes anything. 
`test_` gives bad impression that they are tests, not benchmarks. If conventions are holding us back, we should break those. Anyway, don't have too strong opinion here.